Hunting Queries/Corelight/CorelightAbnormalEmailSubject.yaml

```diff
@@ -0,0 +1,23 @@
+id: e9441d57-39f4-41fb-aaad-d02e47783d1a
+name: Corelight - Abnormal Email Subject
+description: |
+  'Query searches for emails with NON-Ascii characters within the Subject .'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1566
+query: |
+  Corelight
+  | where EventType =~ 'smtp'
+  | where Subject startswith @'\=?utf-16'

```

Comment:
Same here for hasprefix - https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/datatypes-string-operators

Reply:
updated